### PR TITLE
428544/accessbility  portal UI wave

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/views/cdo/create/microchip-search.njk [hunk 1]
```diff
@@ -34,21 +34,13 @@
     {% endif %}
 
     <form method="POST" novalidate autocomplete="off">
```
Comment (jbarnsley10) [inline app/views/cdo/create/microchip-search.njk:36]: Are we going to be missing an h1 from this page now?

File: app/models/cdo/manage/tasks/send-form2.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 const { errorPusherDefault } = require('../../../../lib/error-helpers')
 
 function ViewModel (data, backNav, errors) {
+  const backLink = backNav.backLink === '/' ? `/cdo/manage/cdo/${data.indexNumber}` : backNav.backLink
   this.model = {
-    backLink: backNav.backLink === '/' ? `/cdo/manage/cdo/${data.indexNumber}` : backNav.backLink,
+    backLink,
+    continueLink: backLink + '?action=continue',
```
Comment (jbarnsley10) [inline app/models/cdo/manage/tasks/send-form2.js:7]: I like these for cancel links

File: app/views/cdo/manage/cdo.njk [hunk 3]
```diff
@@ -51,57 +53,29 @@
...
-              {{ task.status }}
-            {% endif %}
-          {% else %}
-          {{ task.status }}
-          {% endif %}
-        </div>
-        </li>
-        {% endfor %}
-      </ul>
+      {{ govukTaskList(model.taskList) }}
```
Comment (jbarnsley10) [inline app/views/cdo/manage/cdo.njk:78]: Ah, I now understand your changes above where all this logic goes into the ViewModel :)

File: app/views/cdo/manage/tasks/send-form2.njk [hunk 4]
```diff
@@ -36,7 +36,7 @@
 
     <form method="POST" novalidate autocomplete="off">
       <span class="govuk-caption-m">Dog {{ model.indexNumber }}</span>
-      <fieldset class="govuk-fieldset" aria-describedby="application-pack-hint">
+      <fieldset class="govuk-fieldset">
```
Comment (jbarnsley10) [inline app/views/cdo/manage/tasks/send-form2.njk:39]: Do we need the described-by to be conditional as in send-application-pack.njk ?

**Review by jbarnsley10 — APPROVED** [review]
A couple of minor questions but all good

File: app/views/cdo/create/microchip-search.njk (hunk 1 above)
Comment (whitewaterdesign) [reply app/views/cdo/create/microchip-search.njk:36]: the h1 has been moved to the govukInput.  Both govukInput and govukFieldset allow you to put the page header into those components.  It resolves the requirement always have a label for input fields.

**Pull request merged**